TaskManager/src/components/TaskCard/TaskCard.jsx

```diff
@@ -0,0 +1,36 @@
+import "./TaskCard.css";
+import Badge from "../Badge/Badge";
+import DateContainer from "../DateContainer/DateContainer";
+import React from "react";
+
+function TaskCard(props) {
+
+  let btnclick = props.id
+
+  const handleClick=() => {
+    console.log("Click from " + props.id);
+    btnclick = props.id + "Clicked !"
+  }
+
+  return (
+		<div className="card-wrapper">
```

Comment:
In loc de taburi ar fi indicat sa folosim spatii.

Reply:
O sa tin cont 🥂